src/aks-preview/azext_aks_preview/_params.py

```diff
@@ -109,6 +109,7 @@ def load_arguments(self, _):
         c.argument('cluster_autoscaler_profile', nargs='+', validator=validate_cluster_autoscaler_profile)
         c.argument('min_count', type=int, validator=validate_nodes_count)
         c.argument('max_count', type=int, validator=validate_nodes_count)
+        c.argument('uptime_sla', action='store_true')
```

Comment:
Question (non-blocking): What is this? Never seen an action before

Reply:
if the value used, then set it to True